test/runtests.jl

```diff
@@ -80,63 +101,72 @@ end
   Base.cumulative_compile_timing(true)
 end
 
-# Used in both Rings/slpolys.jl and StraightLinePrograms/runtests.jl
-const SLP = Oscar.StraightLinePrograms
+println("Making test list")
 
-include("Aqua.jl")
+testlist = []
+push!(testlist, "Aqua.jl")
 
-include("printing.jl")
+push!(testlist, "printing.jl")
 
-include("PolyhedralGeometry/runtests.jl")
-include("Combinatorics/runtests.jl")
+push!(testlist, "PolyhedralGeometry/runtests.jl")
+push!(testlist, "Combinatorics/runtests.jl")
 
-include("GAP/runtests.jl")
-include("Groups/runtests.jl")
+push!(testlist, "GAP/runtests.jl")
+push!(testlist, "Groups/runtests.jl")
 
-include("Rings/runtests.jl")
+push!(testlist, "Rings/runtests.jl")
 
-include("NumberTheory/nmbthy.jl")
+push!(testlist, "NumberTheory/nmbthy.jl")
 
 if Oscar.is_dev
-  include("Experimental/GITFans.jl")
+  push!(testlist, "Experimental/GITFans.jl")
 end
 
 # Will automatically include all experimental packages following our
 # guidelines.
-include("../experimental/runtests.jl")
+push!(testlist, "../experimental/runtests.jl")
+
+push!(testlist, "Experimental/galois.jl")
+push!(testlist, "Experimental/gmodule.jl")
+push!(testlist, "Experimental/ModStdQt.jl")
+push!(testlist, "Experimental/ModStdNF.jl")
+push!(testlist, "Experimental/MatrixGroups.jl")
+push!(testlist, "Experimental/JuLie.jl")
+push!(testlist, "Experimental/ExteriorAlgebra.jl")
 
-include("Experimental/galois.jl")
-include("Experimental/gmodule.jl")
-include("Experimental/ModStdQt.jl")
-include("Experimental/ModStdNF.jl")
-include("Experimental/MatrixGroups.jl")
-include("Experimental/JuLie.jl")
-include("Experimental/ExteriorAlgebra.jl")
+push!(testlist, "Rings/ReesAlgebra.jl")
 
-include("Rings/ReesAlgebra.jl")
+push!(testlist, "Modules/runtests.jl")
 
-include("Modules/runtests.jl")
+push!(testlist, "InvariantTheory/runtests.jl")
 
-include("InvariantTheory/runtests.jl")
+push!(testlist, "AlgebraicGeometry/Schemes/runtests.jl")
+push!(testlist, "AlgebraicGeometry/ToricVarieties/runtests.jl")
+push!(testlist, "AlgebraicGeometry/Surfaces/K3Auto.jl")
 
-include("AlgebraicGeometry/Schemes/runtests.jl")
-include("AlgebraicGeometry/ToricVarieties/runtests.jl")
-include("AlgebraicGeometry/Surfaces/K3Auto.jl")
+push!(testlist, "TropicalGeometry/runtests.jl")
 
-include("TropicalGeometry/runtests.jl")
+push!(testlist, "Serialization/runtests.jl")
 
-include("Serialization/runtests.jl")
+push!(testlist, "StraightLinePrograms/runtests.jl")
 
-include("StraightLinePrograms/runtests.jl")
+# if many workers, distribute tasks across them
+# otherwise, is essentially a serial loop
+@time pmap(x -> include(x), testlist)
```

Comment:
```suggestion
pmap(include, testlist)
```
Or did you want to use `@test` in here on purpose?